src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer/nuxeo-enhanced-viewer-release-notes/nuxeo-enhanced-viewer-release-notes-2-3-0.md

```diff
@@ -0,0 +1,78 @@
+---
+title: NEV 2.3.0
+description: Release notes for Nuxeo Enhanced Viewer 2.3.0
+tree_item_index: 893
+hidden: true
+review:
+  comment: ''
+  date: '2023-02-13'
+  status: ok
+toc: true
+---
+
+{{! multiexcerpt name='nuxeo-enhanced-viewer-updates'}}
+## What's new in NEV 2.3.0
+
+{{#> callout type='warning' heading='Breaking Change'}}
+If you deploy NEV in a self-managed way, the following breaking change applies:
+
+Hazelcast was upgraded from 3.12.x to 4.x, which means that NEV prior to 2.3.x can not interact with NEV after 2.3.x.
+
+If you have deployed NEV in High Availability mode with Hazelcast, you can not do the rolling upgrade when updating to this version. The previous UI stack has to be shutdown before deploying the new version.
+{{/callout}}
+
+### {{> anchor 'summary'}} Summary
+
+<div class="table-scroll">
+<table class="hover">
+<tbody>
+<tr>
+<th colspan="1">NEV Version</th>
+<td colspan="1">2.3.0</td>
+</tr>
+<tr>
+<th colspan="1">Upgraded components</th>
+<td colspan="1">ARender Rendition, ARender Previewer (Nuxeo ARender connector)</td>
+</tr>
+<tr>
+<th colspan="1">ARender Rendition version</th>
+<td colspan="1">nuxeo-arender-document-service-broker 2.3.0, nuxeo-arender-document-renderer 2.3.0, nuxeo-arender-document-converter 2.3.0, nuxeo-arender-document-text-handler 2.3.0</td>
+</tr>
+<tr>
+<th colspan="1">ARender Previewer (Nuxeo ARender connector) version</th>
+<td colspan="1">nuxeo-arender-ui 2.3.0</td>
+</tr>
+</tbody>
+</table>
+</div>
+
+### Improved Compatibility With Microsoft Office Files
+
+This release introduces a new rendering engine that provides better compatibility when viewing complex Microsoft Office Word, Excel and Powerpoint files.
+
+If you are using Nuxeo cloud, this change is transparent.
+
+If you are deploying NEV in a self-managed way, to benefit from it you need to update your `application-security.yml` file related to the TaskConversion module and add the mime types that DirectOffice works with, as shown below:
+
+```
+mimetype:
+  support:
+    msoffice: "${mime.type.msoffice.publisher},${mime.type.msoffice.
+visio},${mime.type.msoffice.rtf},${mime.type.msoffice.project}"
+    libreoffice: "${mime.type.libreoffice.text},${mime.type.libreoffice.
+sheet},${mime.type.libreoffice.presentation},${mime.type.libreoffice.
+graphics}"
+    directoffice: "${mime.type.msoffice.word},${mime.type.msoffice.
+excel},${mime.type.msoffice.powerpoint}"
+```
```

Comment:
This is not true as Direct Office is enabled by default with this version.

Furthermore, the proposed configuration cannot easily applied on the Docker image as it requires to build a custom image which customer may not do.

Reply:
Removed that part after our slack conversation.